Update/ura_010.txt

```diff
@@ -3597,7 +3597,7 @@ void main()
 //r…悲しみの数だけ、人は他人に優しくなれるものだと誰かが言っていた。それはきっと、悲しい思いをしている人の気持ちに対して、素直に同情できるからなのだろう。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…悲しみの数だけ、人は他人に優しくなれるものだと誰かが言っていた。それはきっと、悲しい思いをしている人の気持ちに対して、素直に同情できるからなのだろう。",
-			NULL, "...Someone once said a person can only be kind to others after enduring a number of sorrows. Probably because only then can you truly sympathize with the feelings of sadness in another.", Line_Normal);
+			NULL, "...Someone once said the more sorrows there are, the kinder people can be to others. Probably because only then can you truly sympathize with the feelings of sadness in another.", Line_Normal);
```

Comment:
```suggestion
			NULL, "...Someone once said the more sorrows you experience, the kinder you can be to others. Probably because only then can you truly sympathize with the feelings of sadness in another.", Line_Normal);
```